src/lib.rs

```diff
@@ -429,7 +429,7 @@ mod test {
 
     // Ensure that the CPUs frequency isn't retrieved until we ask for it.
     #[test]
-    #[cfg(not(target_os = "freebsd"))] // In a VM, it'll fail.
+    #[cfg(all(not(target_os = "freebsd"), not(target_arch = "aarch64")))] // In a VM, it'll fail.
```

Comment:
Why this change?

Reply:
I `hw.cpufrequency` seems not working on m1 mac, so I skip it when running unit test.